src/main/java/com/iopipe/IOpipeConfiguration.java

```diff
@@ -499,17 +499,11 @@ public static final IOpipeConfiguration byDefault()
 			// Just use the standard service connction
 			rv.setRemoteConnectionFactory(new ServiceConnectionFactory());
 			
-			// Determine the URI which is used to collect resources, use the
-			// same region as the AWS service if it is supported.
-			String awsregion = Objects.toString(System.getenv("AWS_REGION"),
-				IOpipeConstants.DEFAULT_REGION),
-				origawsregion = awsregion;
-			if (!IOpipeConstants.SUPPORTED_REGIONS.contains(awsregion))
-				awsregion = IOpipeConstants.DEFAULT_REGION;
-			
 			// Setup service URL
-			String surl;
-			rv.setServiceUrl((surl = IOpipeConstants.DEFAULT_SERVICE_URL));
+			String surl = System.getProperty("com.iopipe.collectorurl",
+				Objects.toString(System.getenv("IOPIPE_COLLECTOR_URL"),
+					IOpipeConstants.DEFAULT_SERVICE_URL));
```

Comment:
This reads like it's sending to the default service url and not updating the collector by region -- am I missing something?

Reply:
The `DEFAULT_SERVICE_URL` is set in `IOpipeConstants` and it uses whatever region based on the environment variable (if any) provided by AWS.